File: static/js/GLOBAL_VARIABLES.js


// ----------------- Site URL construction -----------------
// ---Step 1: Specify server and port number
var flip_server = 1     // Choose the server you're using (1, 2, or 3)
var port_num = 10027     // Choose a port number of your choice

// ---Step 2: initialize variables (Don't change any of these)
var baseURL = `http://flip${flip_server}.engr.oregonstate.edu:${port_num}/`
var customers_subpage = baseURL + "customers"
var orders_subpage = baseURL + "orders"
var products_subpage = baseURL + "products"
var departments_subpage = baseURL + "departments"
var seasons_subpage = baseURL + "seasons"



module.exports = {
    baseURL = `http://flip${flip_server}.engr.oregonstate.edu:${port_num}/`,
    "customers_subpage" : baseURL + "customers",
    "orders_subpage" : baseURL + "orders",
    "products_subpage" : baseURL + "products",
    "departments_subpage" : baseURL + "departments",
    "seasons_subpage" : baseURL + "seasons"
}

File: GLOBAL_VARIABLES.py

"""This file holds variables that be accessed from all functions of other .py files, regardless of scope."""

# ----------------- Site URL construction -----------------
# ---Step 1: Specify server and port number (!!! Change these to match your server and port)
flip_server = 3         # Choose your flip server (1, 2, or 3)
port_num = 1028         # Choose a port number of your choice


# ---Step 2: initialize variables (!!! Don't change any of these)
baseUrl = f"http://flip{flip_server}.engr.oregonstate.edu:{port_num}/"
customers_subpage = baseUrl + "customers"
orders_subpage = baseUrl + "orders"
products_subpage = baseUrl + "products"
departments_subpage = baseUrl + "departments"
seasons_subpage = baseUrl + "seasons"
subpages = [baseUrl, customers_subpage, orders_subpage, products_subpage, departments_subpage, seasons_subpage]
